masterlist.yaml

```diff
@@ -8654,16 +8760,12 @@ plugins:
         <<: *dirtyPlugin
         itm: 306
         udr: 10
-  - name: 'Requiem - Dragonborn.esp'
-    dirty:
-      - crc: 0x740bb9f0
-        <<: *dirtyPlugin
-        itm: 3
-  - name: 'Requiem - Hearthfires.esp'
-    dirty:
-      - crc: 0xa3a78f22
-        <<: *dirtyPlugin
-        itm: 1
+  # Must be loaded directly after Requiem.esp
```

Comment:
It's codified as best as possible in the priority value, isn't it? We don't have a better way of saying "load directly after".